src/build.js

```diff
@@ -17,13 +17,14 @@ const walk = async dir => {
     const stat = fs.statSync(filePath)
 
     if (stat.isDirectory()) {
-      tree[propName] = await walk(filePath)
+      tree[propName] = await walk(filePath, wombatUrl)
     }
 
     if (stat.isFile()) {
-      const content = fs.readFileSync(filePath, 'utf8')
+      let content = fs.readFileSync(filePath, 'utf8')
```

Comment:
Mysle ze warto gdzie sie da nie uzywac letow
```suggestion
      const rawContent = fs.readFileSync(filePath, 'utf8')
      const content = rawContent.replace(/\{\{wombatUrl\}\}/g, wombatUrl)
```

Reply:
done